### DISCUSSION TITLE
GitHub UI marks commits made earlier this year are shown as `last year`

### DISCUSSION THREAD

**Discussion by crazoter** [community/community#41884] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**Description**
Commits that were made earlier this year (e.g. May) are dated as `last year`, which is misleading:
![image](https://user-images.githubusercontent.com/9276766/208016675-e820130d-21dd-48c2-928f-373c09c59a46.png)

**Preferred resolution**
Change to `this year`, or put the month etc.

**Comment by ladipro** [comment]
It is January 24th 2023 today and a blame is telling me that a commit made on Aug 19th **2021** happened "last year".

![image](https://user-images.githubusercontent.com/12206368/214267502-20566358-ccd7-4db1-b947-41acaa92eb75.png)

**Comment by luangong** [comment]
It’s February 7th, 2023 3:33 AM GMT+8 (China Standard Time) and commits made on January 10th and on 11th are both shown as “last year,” but two commits made on January 7th and on January 13th are correctly shown as “Jan 7” and as “3 weeks ago,” respectively. It’s really weird.

<img width="680" alt="github_commit_incorrect_relative_time" src="https://user-images.githubusercontent.com/1857158/217062296-3a575b77-daee-498b-89c8-f049310d6b8a.png">

Upon further observation, the bug appears even when two commits are both made on Jan 13th, 2023.

<img width="512" alt="github_commit_incorrect_relative_time_history" src="https://user-images.githubusercontent.com/1857158/217066875-06e163e5-b1a7-44fa-9c5c-08388b91c45f.png">

<img width="734" alt="github_commit_incorrect_relative_time_dom" src="https://user-images.githubusercontent.com/1857158/217066973-1d0c6858-d485-4eef-a98b-9e6b25f39900.png">

This bug seems to be triggered when a commit is made within the past 3-4 weeks.

**Comment by mattandrews** [comment]
I'm seeing this for commits made last month (eg. January 2023) being marked as "last year":

![image](https://user-images.githubusercontent.com/394376/217219563-5ba83c2e-6f65-4d29-824d-48129ebcc882.png)

![image](https://user-images.githubusercontent.com/394376/217219585-8837b394-af5e-4406-9220-69ce6af7a275.png)

**Reply by bxjw** [reply]
Came here to say the same (with, coincidentally, the same dates), noticed it while I was adding a new tag this morning:
![image](https://user-images.githubusercontent.com/15195793/217242636-2d78e173-761b-43e0-9445-61528e156752.png)

**Comment by alistairewj** [comment]
Us as well. Looking at a very active repo like pytorch, improperly reporting commits as last year:

- Starts Jan 8th: https://github.com/pytorch/pytorch/commits/master?after=513b5da3573ffb542ac056dbc6142780a6fb43a5+1035&branch=master&qualified_name=refs%2Fheads%2Fmaster
- Ends Jan 12th: https://github.com/pytorch/pytorch/commits/master?before=513b5da3573ffb542ac056dbc6142780a6fb43a5+896&branch=master&qualified_name=refs%2Fheads%2Fmaster

**Comment by bxjw** [comment]
Is there some way to make this a bug label rather than product feedback?

**Comment by bxjw** [comment]
Update: I submitted a bug ticket this morning and they have fixed it allegedly!
